pkg/cmd/openshift-kube-apiserver/cmd.go

```diff
@@ -70,14 +70,21 @@ func NewOpenShiftKubeAPIServerServerCommand(name, basename string, out, errout i
 				}
 				klog.Fatal(err)
 			}
+			// When no error is returned, always return with zero exit code.
+			// This is here to make sure the container that run apiserver won't get accidentally restarted
+			// when the pod runs with restart on failure.
 		},
 	}
 
 	flags := cmd.Flags()
 	// This command only supports reading from config
 	flags.StringVar(&options.ConfigFile, "config", "", "Location of the master configuration file to run from.")
-	cmd.MarkFlagFilename("config", "yaml", "yml")
-	cmd.MarkFlagRequired("config")
+	if err := cmd.MarkFlagFilename("config", "yaml", "yml"); err != nil {
+		klog.Warningf("Error setting config flag: %v", err)
```

Comment:
what does it mean to get an error here?

Reply:
for example the flag does not exists (or is removed). in that case this throws error. goland is unhappy when you don't catch these and I think in general we should catch all errors ;-)

Reply:
was wondering whether a panic makes more sense here.

Reply:
i don't think this is panic worthy, @deads2k ?

Reply:
We don't worry about that anywhere, it's safe to ignore that one error. Not even warning worthy.